ml_metadata/__init__.py

```diff
@@ -29,5 +29,13 @@
 
 except ImportError as err:
   import sys
-  sys.stderr.write('Error importing: {}'.format(err))
+
+  sys.stderr.write("Error importing: {}".format(err))
 # pylint: enable=g-import-not-at-top
+
+__all__ = [
+  "downgrade_schema",
+  "ListOptions",
+  "MetadataStore",
+  "OrderByField",
+]
```

Comment:
This isn't in scope for this PR, but the try/except block above doesn't work. In the event of an `ImportError`, this module won't be able to export these symbols. I can't really tell what is intended by the try/except block swallowing any import errors, but maybe this is something we can clean up in a future PR.

Reply:
This seems to have been [present since before we started working on the repo](https://github.com/google/ml-metadata/blame/aa9c24977144a05616ffab0083f54f0224dc09aa/ml_metadata/__init__.py#L19). 

I agree with you that we can possibly revisit this in a future PR